<?xml version="1.0" encoding="UTF-8"?>
<saml2p:Response
  Destination="http://1.lvh.me/users/auth/mpassid/callback"
  ID="_3791ae7f525515749a653f55b0fc75fb"
  InResponseTo="_c2e7e3a2-683e-4c0a-8895-e0e8f1caf4d8"
  IssueInstant="2019-08-15T08:36:11.002Z"
  Version="2.0"
  xmlns:saml2p="urn:oasis:names:tc:SAML:2.0:protocol">
  <saml2:Issuer xmlns:saml2="urn:oasis:names:tc:SAML:2.0:assertion">https://mpass-proxy-test.csc.fi/idp/shibboleth</saml2:Issuer>
  <saml2p:Status>
    <saml2p:StatusCode Value="urn:oasis:names:tc:SAML:2.0:status:Requester">
      <saml2p:StatusCode Value="urn:oasis:names:tc:SAML:2.0:status:AuthnFailed" />
    </saml2p:StatusCode>
    <saml2p:StatusMessage>An error occurred.</saml2p:StatusMessage>
  </saml2p:Status>
</saml2p:Response>
